usecases/cluster/mocks/node_selector.go

```diff
@@ -0,0 +1,54 @@
+//                           _       _
+// __      _____  __ ___   ___  __ _| |_ ___
+// \ \ /\ / / _ \/ _` \ \ / / |/ _` | __/ _ \
+//  \ V  V /  __/ (_| |\ V /| | (_| | ||  __/
+//   \_/\_/ \___|\__,_| \_/ |_|\__,_|\__\___|
+//
+//  Copyright © 2016 - 2024 Weaviate B.V. All rights reserved.
+//
+//  CONTACT: hello@weaviate.io
+//
+
+package mocks
+
+import "sort"
+
+type memberlist struct {
+	// nodes include the node names only
+	nodes []string
+}
+
+func (m memberlist) StorageCandidates() []string {
+	sort.Strings(m.nodes)
+	return m.nodes
+}
+
+func (m memberlist) NonStorageNodes() []string {
+	return []string{}
+}
+
+func (m memberlist) SortCandidates(nodes []string) []string {
+	sort.Strings(m.nodes)
+	return nodes
```

Comment:
hmm this method doesn't sort the nodes list passed as argument it only sorts the `nodes` list that is in `memberlist` struct... what is this method meant to do? can we remove it?

Reply:
it's mocking the interface, we have to keep it but good catch will handle that sorting because also needed